Gems/LmbrCentral/Code/Source/Shape/EditorCapsuleShapeComponent.h

```diff
@@ -38,22 +45,43 @@ namespace LmbrCentral
 
         static void GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible);
 
-        // EditorComponentBase
+        // EditorComponentBase overrides ...
         void BuildGameEntity(AZ::Entity* gameEntity) override;
 
+        // AZ::TransformNotificationBus overrides ...
+        void OnTransformChanged(const AZ::Transform& local, const AZ::Transform& world) override;
+
     private:
         AZ_DISABLE_COPY_MOVE(EditorCapsuleShapeComponent)
 
-        // AzFramework::EntityDebugDisplayEventBus
+        // AzFramework::EntityDebugDisplayEventBus overrides ...
         void DisplayEntityViewport(
             const AzFramework::ViewportInfo& viewportInfo,
             AzFramework::DebugDisplayRequests& debugDisplay) override;
 
+        // AzToolsFramework::CapsuleManipulatorRequestBus overrides ...
+        float GetHeight() const override;
+        void SetHeight(float height) override;
+        AZ::Quaternion GetRotationOffset() const override;
+
+        // AzToolsFramework::RadiusManipulatorRequestBus overrides ...
+        float GetRadius() const override;
+        void SetRadius(float radius) override;
+
+        // AzToolsFramework::ShapeManipulatorRequestBus overrides ...
+        AZ::Vector3 GetTranslationOffset() const override;
+        void SetTranslationOffset(const AZ::Vector3& translationOffset) override;
+        AZ::Transform GetManipulatorSpace() const override;
+
         AZ::Crc32 ConfigurationChanged();
         void ClampHeight();
         void GenerateVertices();
 
         CapsuleShape m_capsuleShape; ///< Stores underlying capsule representation for this component.
         ShapeMesh m_capsuleShapeMesh; ///< Buffer to hold index and vertex data for CapsuleShape when drawing.
+
+        using ComponentModeDelegate = AzToolsFramework::ComponentModeFramework::ComponentModeDelegate;
+        ComponentModeDelegate m_componentModeDelegate; /**< Responsible for detecting ComponentMode activation
+                                                         *  and creating a concrete ComponentMode.*/
```

Comment:
nit: I'd update this comment to `//!<` style if possible (instead of a multi-line comment)

Reply:
Updated